include/splinterdb/platform_linux/public_platform.h

```diff
@@ -78,3 +78,7 @@ typedef FILE platform_log_handle;
 void
 platform_set_log_streams(platform_log_handle *info_stream,
                          platform_log_handle *error_stream);
+
+// Reset ouput log streams to their default settings.
+void
+platform_reset_log_streams(void);
```

Comment:
Used to re-set output streams to initial (default) setting in case the env-var level is -not- what the test-case asks for.